Microsoft.Toolkit/Microsoft.Toolkit.csproj

```diff
@@ -25,5 +25,8 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.4'">
     <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
+  </ItemGroup>
```

Comment:
This package reference shouldn't be unconditional here, as it's not needed on NS2.1 and .NET 5.
This should only be references on targets below them.

Reply:
Care to propose a suggestion?

Reply:
Are we targeting below NS2.0? I thought this was part of NS2.0 as well? As it seems like we only include if for 1.4 in the Diagnostics package here: https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Diagnostics/Microsoft.Toolkit.Diagnostics.csproj

Reply:
Ah, Michael beat me to it ahah 😄
Yeah we can just copy the same template we're using in other packages such as the `Diagnostics` one, and simply add that package reference for the .NET Standard 1.4 target and leave the others free. Then one day I'll just nuke all of them and make life easier for everyone, but this is not that day just yet, maybe in 8.0 ahah

Reply:
Updated!